pkgs/games/quakespasm/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, SDL, fetchurl, gzip, libvorbis, libmad }:
+stdenv.mkDerivation rec {
+  name = "quakespasm-${version}";
+  majorVersion = "0.92";
+  version = "${majorVersion}.1";
+  platform = "amd64";
```

Comment:
Why do you need to define `platform` here? I think this can be removed as it doesn't do anything (except being an environment variable during the build).